banking/src/main/java/rocks/kata/DatabaseClasses/AccountOperationDatabaseFormatter.java

```diff
@@ -0,0 +1,21 @@
+package rocks.kata.DatabaseClasses;
+
+import java.util.List;
+
+import rocks.kata.AccountOperation;
+
+public class AccountOperationDatabaseFormatter {
+
+    public String createFormattedStringForAccountOperations(List<AccountOperation> listOFOperationsForAccount) {
```

Comment:
zamiast `listOFOperationsForAccount` wystarczy `accountOperations` albo po prostu `operations` - to, ze to lista widac po typie, a gdyby sie zmienil, to nazwa parametru bedzie klamac